app/src/main/res/values/strings.xml

```diff
@@ -491,43 +491,43 @@
     <string name="ct_bs_app_info_error">App Info not found</string>
 
     <!-- Home screen -->
-    <string name="firewall_card_status_inactive">Disabled</string>
+    <string name="firewall_card_status_inactive">Off</string>
     <string name="firewall_card_status_active">%1$s apps</string>
 
     <string name="firewall_card_text_active">%1$s blocked\n%2$s bypassed\n%3$s excluded\n%4$s lockdown</string>
-    <string name="firewall_card_text_inactive">Enable firewall mode.</string>
+    <string name="firewall_card_text_inactive">Turn on firewall mode.</string>
     <string name="firewall_card_configure_btn">block apps</string>
 
-    <string name="dns_card_connected_status_failure">Disabled</string>
+    <string name="dns_card_connected_status_failure">Off</string>
 
     <string name="dns_card_latency_active">Latency: %1$sms</string>
-    <string name="dns_card_latency_inactive">Enable DNS mode.</string>
+    <string name="dns_card_latency_inactive">Turn on DNS mode.</string>
 
-    <string name="apps_card_ips_count">%1$s IP &amp; Port rules</string>
+    <string name="apps_card_ips_count">%1$s IP and port rules</string>
 
     <string name="hsf_start_btn_state">start</string>
     <string name="hsf_stop_btn_state">stop</string>
 
     <string name="hsf_uptime">(active since %1$s; protos: %2$s)</string>
     <string name="hsf_downtime">(inactive since %1$s)</string>
 
-    <string name="hsf_vpn_dialog_header">Attention!</string>
-    <string name="hsf_vpn_dialog_message">RethinkDNS needs permission to set up a local VPN to encrypt the device\'s DNS requests, implement a network monitor, and enforce firewall rules. \n\nRethinkDNS does not collect any information or spy on you in any way.</string>
+    <string name="hsf_vpn_dialog_header">Action needed</string>
```

Comment:
`Attention!` with the `!` is meant for the user to seriously pay attention to this dialog....

Reply:
Action needed is used elsewhere. Can add an exclamation point to the actual message.

Reply:
Here, the title is `Attention!` because the app wouldn't function at all unless the user takes action. And the user must pay attention to the text of the dialog before taking any action / no-action. Whereas, for `Action needed`, it is enough if the user just looks at the *action buttons*. Subtle but that's how we intended it.